base/src/main/java/com/smartdevicelink/managers/screen/choiceset/KeyboardAutocompleteCompletionListener.java

```diff
@@ -32,11 +32,20 @@
 
 package com.smartdevicelink.managers.screen.choiceset;
 
+import java.util.List;
+
 public interface KeyboardAutocompleteCompletionListener {
 
 	/**
 	 * This listener is called when you wish to update your autocomplete text in response to the user's input
 	 * @param updatedAutoCompleteText - The new autocomplete text to use
 	 */
+	@Deprecated
```

Comment:
Since the method has changed can we add a "use (@link#method)" to the new method in the docs?